Dockerfile

```diff
@@ -4,14 +4,24 @@ FROM ubuntu:23.10
 RUN apt-get update && \
     apt-get install -y \
     curl \
+    udev \
     wget \
-    nano
+    nano \
+    build-essential \
+    symlinks \
+    expect \
+    git 
+#Copy current git repo into project directory
+RUN git clone https://github.com/JGoard/jocktos.git && cd ..
```

Comment:
This isn't doing anything, the container is already mounted to the hosts workspace folder